ephios/core/forms/users.py

```diff
@@ -20,43 +20,82 @@
 from ephios.core.consequences import WorkingHoursConsequenceHandler
 from ephios.core.models import QualificationGrant, UserProfile
 from ephios.core.widgets import MultiUserProfileWidget
+from ephios.extra.mixins import PermissionFormMixin
+from ephios.extra.permissions import PermissionField
 from ephios.extra.widgets import CustomDateInput
 
 
-class GroupForm(ModelForm):
+class GroupForm(PermissionFormMixin, ModelForm):
+    can_view_past_event = PermissionField(
+        label=_("Can view past events"), permissions=["core.view_past_event"], required=False
+    )
+
+    is_planning_group = PermissionField(
+        label=_("Can add events"),
+        permissions=["core.add_event", "core.delete_event"],
+        required=False,
+    )
     publish_event_for_group = ModelMultipleChoiceField(
         label=_("Can publish events for groups"),
         queryset=Group.objects.all(),
         required=False,
         help_text=_("Choose groups that this group can make events visible for."),
         widget=Select2MultipleWidget,
     )
-    can_view_past_event = BooleanField(label=_("Can view past events"), required=False)
-    can_add_event = BooleanField(label=_("Can add events"), required=False)
-    can_manage_user = BooleanField(
-        label=_("Can manage users"),
+    decide_workinghours_for_group = ModelMultipleChoiceField(
+        label=_("Can decide working hours for groups"),
+        queryset=Group.objects.all(),
+        required=False,
+        help_text=_(
+            "Choose groups that the group you are currently editing can decide whether to grant working hours for."
+        ),
+        widget=Select2MultipleWidget,
+    )
+
+    is_hr_group = PermissionField(
+        label=_("Can edit users"),
         help_text=_("If checked, users in this group can view, add, edit and delete users."),
```

Comment:
add "users in this group can also manage group memberships regarding any of their own groups." or smth like that.